hi-ml-azure/src/health_azure/utils.py

```diff
@@ -1046,6 +1053,7 @@ def get_run_file_names(run: Run, prefix: str = "") -> List[str]:
     :return: A list of paths within the Run's container
     """
     all_files = run.get_file_names()
+    print(f"Selecting files with prefix {prefix}")
```

Comment:
Did you leave this print statement in by accident?

Reply:
No, I made it a bit nicer and left it on purpose - I found it's useful to know which prefix is used to select among run files